codegen/src/test/resources/software/amazon/awssdk/codegen/poet/auth/scheme/query-endpoint-auth-params-auth-scheme-default-params.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.query.auth.scheme.internal;
+
+import java.util.Optional;
+import software.amazon.awssdk.annotations.Generated;
+import software.amazon.awssdk.annotations.SdkInternalApi;
+import software.amazon.awssdk.services.query.auth.scheme.QueryAuthSchemeParams;
+import software.amazon.awssdk.utils.Validate;
+
+@Generated("software.amazon.awssdk:codegen")
+@SdkInternalApi
+public final class DefaultQueryAuthSchemeParams implements QueryAuthSchemeParams {
+  private final String operation;
+
+  private final String region;
+
+  private final Boolean useDualStackEndpoint;
+
+  private final Boolean useFipsEndpoint;
+
+  private final String endpointId;
+
+  private final Boolean defaultTrueParam;
+
+  private final String defaultStringParam;
+
+  private final String deprecatedParam;
+
+  private final Boolean booleanContextParam;
+
+  private final String stringContextParam;
+
+  private final String operationContextParam;
+
+  private DefaultQueryAuthSchemeParams(Builder builder) {
+    this.operation = Validate.paramNotNull(builder.operation, "operation");
+    this.region = builder.region;
+    this.useDualStackEndpoint = builder.useDualStackEndpoint;
+    this.useFipsEndpoint = builder.useFipsEndpoint;
+    this.endpointId = builder.endpointId;
+    this.defaultTrueParam = builder.defaultTrueParam;
```

Comment:
Shouldn't "defaultTrueParam" lead to accessor defaulting to returning `true` when not set in the builder?